Documentation/etcd-mixin/mixin.libsonnet

```
@@ -8,6 +8,19 @@
       {
         name: 'etcd',
         rules: [
+          {
+            alert: 'etcdMembersDown',
+            expr: |||
+              max(sum(up{%(etcd_selector)s} == bool 0) by (job) OR count(sum(rate(etcd_network_peer_sent_failures_total{%(etcd_selector)s}[3m])) by (To,job,endpoint) > 0.01) by (job,endpoint) > 0) by (job)
```

I'd propose to use the upstream formatting for alerting rules: 
```suggestion
              max by (job) (
                sum by (job) (up{%(etcd_selector)s} == bool 0)
              or
                count by (job, endpoint) (
                  sum by (To,job,endpoint) (rate(etcd_network_peer_sent_failures_total{%(etcd_selector)s}[3m])) > 0.01
                )
              ) > 0
```

I think we need to move the last `> 0` out of the `max()`, right?

---

Updated with better tests - it was not properly flagging the 3m delay.